src/CTA.FeatureDetection.Common/Models/Features/Base/Feature.cs

```diff
@@ -34,7 +34,7 @@ public override bool Equals(object obj)
 
         public override int GetHashCode()
         {
-            return (Name != null ? StringComparer.OrdinalIgnoreCase.GetHashCode(Name) : 0);
+            return HashCode.Combine((Name != null ? StringComparer.OrdinalIgnoreCase.GetHashCode(Name) : 0));
```

Comment:
HashCode.Combine can be used like this:
HashCode.Combine(Name). 

If name is null, it shouldn't throw an error

Reply:
As mentioned above, thanks.